velox/exec/tests/utils/PlanBuilder.cpp

```diff
@@ -1524,6 +1524,94 @@ PlanBuilder& PlanBuilder::window(
       sortingOrders,
       windowNames,
       windowNodeFunctions,
+      false,
+      planNode_);
+  return *this;
+}
+
+PlanBuilder& PlanBuilder::streamingWindow(
```

Comment:
@JkSelf : There isn't a need of duplicating the code.

Rename original window(...) function to commonWindow(...) and invoke that
in new window(...) and streamingWindow() signatures with the original parameters and correct value of inputSorted true or false based on window style.

Reply:
@aditi-pandit @mbasmanova Introduce a commonWindow method to create the WindowNode. Then, modify the window() and streamingWindow() functions to invoke commonWindow() and provide the inputSorted value as a parameter.